src/uikit.cs

```diff
@@ -11742,6 +11935,10 @@ interface UIWindow {
 		[NullAllowed, Export ("canResizeToFitContent")]
 		bool CanResizeToFitContent { get; [Bind ("setCanResizeToFitContent:")] set; }
 
+		[Deprecated (PlatformName.iOS, 26, 0, message: "Use the 'UIWindowScene' overload instead.")]
+		[Deprecated (PlatformName.TvOS, 26, 0, message: "Use the 'UIWindowScene' overload instead.")]
+		[Deprecated (PlatformName.MacCatalyst, 26, 0, message: "Use the 'UIWindowScene' overload instead.")]
+		[DesignatedInitializer]
 		[Export ("initWithFrame:")]
 		NativeHandle Constructor (CGRect frame);
```

Comment:
I think we'll have to update our templates, because they use this constructor (and we have tests verifying that the templates are warning-free)